src/subscribeToStateUpdates.tsx

```diff
@@ -11,6 +7,10 @@ import isShallowEqual from './utils/isShallowEqual';
 type SubscriberGenerator = (WrappedComponent: ComponentType<any>) => (props: any) => JSX.Element;
 
 /**
+ * @deprecated
```

Comment:
can the `deprecated` tag and message be moved to the end? I think the docs get formatted better that way because it's able to pick out where the deprecation message ends more easily

Reply:
sure!